### DISCUSSION TITLE
Says "Built in firewall kill switch to allow traffic only with needed the VPN servers and LAN devices" Any set up required?

### DISCUSSION THREAD

**Discussion by aprillia99** [qdm12/gluetun#358] [Q&A] [answered]
Is the kill switch defaulted to be on? is there anything we have to set up or enable?

**Comment by qdm12** [comment, marked as answer]
No, just don't disable it 🙈 (as in don't use `FIREWALL=off`)

It uses `iptables` under the hood to block everything not going through the VPN tunnel. There is a [whole package I wrote in Go](https://github.com/qdm12/gluetun/tree/master/internal/firewall) to do that, if you want to dig into which rules is set but it's a bit complex even for me to be honest 😅

**Reply by aprillia99** [reply]
I don't know if you have seen the level of questions I'm asking on here, but I'm sure I would look at it and leave more confused than when I started. 

Thank you for your work on these projects! I really appreciate the dedication and follow up even on the silly questions!
